.gitignore

```diff
@@ -125,3 +125,6 @@ benchmark_*.svg
 
 ## Kind
 .kube
+
+## Helm Cache
+cilium/Library/
```

Comment:
Can we select somewhere else, or clean up after tests?

Reply:
I added a [temp directory](https://github.com/DataDog/integrations-core/pull/11193/files#diff-caca836fda40c77b052f8036ccaba12b61ac665f51474c309ce39846f72da24fR111) for the helm cache files, which should get removed when the E2E is torn down. Let me know if this is the right approach for using `temp_dir`.